numpy/tests/test_gitignore.py

```diff
@@ -0,0 +1,15 @@
+import subprocess as sp
+import pytest
+
+
+def test_no_untracked_files():
+    fd_1 = sp.run(["git", "status", "-u", "--porcelain"], stdout=sp.PIPE,
+                  stderr=sp.PIPE, check=True).stdout.decode(encoding='UTF-8',
+                                                            errors='strict')\
```

Comment:
You can use universal_newline to get a decoded string out. I am not a big fan of `\` line continuations, please use a temporary variable.

Reply:
Done!